src/components/result-ui/summary.js

```diff
@@ -58,31 +54,24 @@ const Summary = ({data, focusPath}) => {
     <ul>
       {resourceObjects.map((resourceObject, i) => {
         const type = resourceObject.getType(), id = resourceObject.getID();
-        const attributes = Object.entries(resourceObject.getAttributes()).filter(([fieldPath]) => !focus || fieldPath === focus);
+        const attributes = Object.entries(resourceObject.getAttributes()).filter(([fieldPath]) => !focusPath || fieldPath === focusPath);
```

Comment:
it might be more performant to use this to set a class and hide with css instead, I think the big multi row re-render is why it is behaving slowly.

Reply:
Good tip. This is probably better anyway if we wanted to do any kind of animation.